src/runtime/node/punycode/index.ts

```diff
@@ -0,0 +1,8 @@
+import type punycode from "node:punycode";
+
+import * as _punycode from "punycode/";
```

Comment:
is the `/` suffix required here?

Reply:
Unfortunately, yes, since `punycode` is a node built-in module :(